src/graph/depth_order.rs

```diff
@@ -130,8 +125,7 @@ fn visit_by_depth(graph: &Graph,
                   maybe_captured_mouse: Option<NodeIndex>,
                   maybe_captured_keyboard: Option<NodeIndex>,
                   depth_order: &mut Vec<Visitable>,
-                  floating_deque: &mut Vec<NodeIndex>)
-{
+                  floating_deque: &mut Vec<NodeIndex>) {
```

Comment:
For multi-line arguments, I generally prefer to have the brace on the next line. This might just be a personal preference thing though.